### ISSUE TITLE
The padding get none in conv op

### ISSUE THREAD

**Issue by SunAriesCN** [ethereon/caffe-tensorflow#130]
When I converted the SequeezeNet,  I got a modules as following:
`        (self.feed('fire9_expand1x1', 
                   'fire9_expand3x3')
             .concat(3, name='fire9_concat')
             .conv(1, 1, 1000, 1, 1, padding=None, name='conv10')
             .avg_pool(14, 14, 1, 1, padding='VALID', name='pool10')
             .softmax(name='prob'))
`
And I just found that is caused by the limitation of tensorflow, in Convolution padding, comparing with caffe. However,  tensorfow lastest version has the tf.pad function may resolve this problem. Have some one to help us improve this bug.
